Thanks for the reactivity @mjwitte.

I suggest removing this build warning

```
/home/julien/Software/Others/EnergyPlus/src/EnergyPlus/HVACUnitaryBypassVAV.cc: In function ‘void EnergyPlus::HVACUnitaryBypassVAV::GetCBVAV()’:
/home/julien/Software/Others/EnergyPlus/src/EnergyPlus/HVACUnitaryBypassVAV.cc:534:14: warning: unused variable ‘EquipNum’ [-Wunused-variable]
   static int EquipNum( 0 ); // local do loop index for equipment listed for a zone
```

--- a/src/EnergyPlus/HVACUnitaryBypassVAV.cc
+++ b/src/EnergyPlus/HVACUnitaryBypassVAV.cc
@@ -201,6 +201,7 @@ namespace HVACUnitaryBypassVAV {
 	Real64 SaveCompressorPLR( 0.0 ); // Holds DX compressor PLR from active DX coil
 	Real64 TempSteamIn( 100.0 ); // steam coil steam inlet temperature
 	Array1D_bool CheckEquipName;
+	bool GetInputFlag( true ); // Flag set to make sure you get input once
 
 	// SUBROUTINE SPECIFICATIONS FOR MODULE
 
@@ -208,6 +209,26 @@ namespace HVACUnitaryBypassVAV {
 	Array1D< CBVAVData > CBVAV;
 
 	// Functions
+
+	void
+	clear_state()
+	{
+		CBVAV.deallocate();
+		NumCBVAV = 0;
+		CompOnMassFlow = 0.0;
+		OACompOnMassFlow = 0.0;
+		CompOffMassFlow = 0.0;
+		OACompOffMassFlow = 0.0;
+		CompOnFlowRatio = 0.0;
+		CompOffFlowRatio = 0.0;
+		FanSpeedRatio = 0.0;
+		BypassDuctFlowFraction = 0.0;
+		PartLoadFrac = 0.0;
+		SaveCompressorPLR = 0.0;
+		TempSteamIn = 100.0;
+		CheckEquipName.deallocate();
+		GetInputFlag = true;
+	}
 
 	void
 	SimUnitaryBypassVAV(
@@ -250,7 +271,6 @@ namespace HVACUnitaryBypassVAV {
 
 		// SUBROUTINE LOCAL VARIABLE DECLARATIONS:
 		int CBVAVNum; // Index of CBVAV system being simulated
-		static bool GetInputFlag( true ); // First time, input is "gotten"
 		Real64 OnOffAirFlowRatio; // Ratio of compressor ON airflow to average airflow over timestep
 		Real64 QUnitOut; // Sensible capacity delivered by this air loop system
 		Real64 QZnLoad; // Zone load required by all zones served by this air loop system

--- a/src/EnergyPlus/HVACUnitaryBypassVAV.cc
+++ b/src/EnergyPlus/HVACUnitaryBypassVAV.cc
@@ -472,8 +472,6 @@ namespace HVACUnitaryBypassVAV {
 		using NodeInputManager::GetOnlySingleNode;
 		using DataZoneEquipment::ZoneEquipConfig;
 		using DataZoneEquipment::ZoneEquipList;
-		using DataZoneEquipment::AirDistUnit_Num;
-		using DataZoneEquipment::DirectAir_Num;
 		using BranchNodeConnections::SetUpCompSets;
 		using BranchNodeConnections::TestCompSet;
 		using DataZoneControls::TempControlledZone;
@@ -531,7 +529,6 @@ namespace HVACUnitaryBypassVAV {
 		Array1D_string cNumericFields( 11 ); // Numeric field names
 		Array1D_bool lAlphaBlanks( 19 ); // Logical array, alpha field input BLANK = .TRUE.
 		Array1D_bool lNumericBlanks( 11 ); // Logical array, numeric field input BLANK = .TRUE.
-		static int EquipNum( 0 ); // local do loop index for equipment listed for a zone
 		int HeatCoilInletNodeNum; // Heating coil air inlet node number
 		int HeatCoilOutletNodeNum; // Heating coil air outlet node number
 		int SteamIndex; // steam coil index

--- a/src/EnergyPlus/HVACUnitaryBypassVAV.hh
+++ b/src/EnergyPlus/HVACUnitaryBypassVAV.hh
@@ -388,8 +388,12 @@ namespace HVACUnitaryBypassVAV {
 
 	// Object Data
 	extern Array1D< CBVAVData > CBVAV;
+	extern bool GetInputFlag; // Flag set to make sure you get input once
 
 	// Functions
+
+	void
+	clear_state();
 
 	void
 	SimUnitaryBypassVAV(

--- a/tst/EnergyPlus/unit/Fixtures/EnergyPlusFixture.cc
+++ b/tst/EnergyPlus/unit/Fixtures/EnergyPlusFixture.cc
@@ -146,6 +146,7 @@
 #include <EnergyPlus/HVACManager.hh>
 #include <EnergyPlus/HVACSingleDuctInduc.hh>
 #include <EnergyPlus/HVACStandAloneERV.hh>
+#include <EnergyPlus/HVACUnitaryBypassVAV.hh>
 #include <EnergyPlus/HVACUnitarySystem.hh>
 #include <EnergyPlus/HVACVariableRefrigerantFlow.hh>
 #include <EnergyPlus/HybridModel.hh>
@@ -374,6 +375,7 @@ namespace EnergyPlus {
 		HVACManager::clear_state();
 		HVACSingleDuctInduc::clear_state();
 		HVACStandAloneERV::clear_state();
+		HVACUnitaryBypassVAV::clear_state();
 		HVACUnitarySystem::clear_state();
 		HVACVariableRefrigerantFlow::clear_state();
 		HybridModel::clear_state();

